kqueen_ui/blueprints/manager/views.py

```diff
@@ -168,42 +168,72 @@ class MemberCreate(KQueenView):
     methods = ['GET', 'POST']
 
     def handle(self, organization_id):
-        form = MemberCreateForm()
+        form_cls = MemberCreateForm
+        auth_options = app.config.get('AUTH_OPTIONS', {})
+        if auth_options:
+            auth_choices = []
+            for name, options in auth_options.items():
+                choice = (name, options.get('label', name))
+                auth_choices.append(choice)
+            field_kw = {
+                'auth_method': {
+                    'type': 'select',
+                    'label': 'Authentication Method',
+                    'choices': auth_choices,
+                    'validators': {
+                        'required': True
+                    }
+                }
+            }
+            form_cls.append_fields(field_kw)
+        form = form_cls()
         if form.validate_on_submit():
+            auth_method = 'local'
+            notify = True
```

Comment:
its redundant coz in a few lines below u already set default value

Reply:
Not really, if you mean inside the `if` condition, that code is not executed if you didn't specify additional AUTH_OPTIONS in config, so the code would fail with `notify` not defined

Reply:
oh, okok